src/Snapshooter/Exceptions/NoSnapshotFoundException.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Snapshooter.Exceptions
+{
+    /// <summary>
+    /// Exception thrown if something went wrong during snapshot assertion.
+    /// </summary>
+    public class NoSnapshotFoundException : SnapshotTestException
```

Comment:
How about SnapshotNotFoundException instead of NoSnapshotFoundexception

Reply:
I agree, that name is better!